mmv1/third_party/terraform/website/docs/r/container_cluster.html.markdown

```diff
@@ -186,13 +186,14 @@ set this to a value of at least `1`, alongside setting
 `remove_default_node_pool` to `true`.
 
 * `ip_allocation_policy` - (Optional) Configuration of cluster IP allocation for
-VPC-native clusters. Adding this block enables [IP aliasing](https://cloud.google.com/kubernetes-engine/docs/how-to/ip-aliases),
-making the cluster VPC-native instead of routes-based. Structure is [documented
+VPC-native clusters. If this block is unset during creation, it will be set by the GKE backend. 
+Adding this block enables [IP aliasing](https://cloud.google.com/kubernetes-engine/docs/how-to/ip-aliases),
```

Comment:
"Adding this block enables [IP aliasing](https://cloud.google.com/kubernetes-engine/docs/how-to/ip-aliases), making the cluster VPC-native instead of routes-based" is outdated

Reply:
Swore I removed this line. I'll update it in the `main` PR